avr/cores/megacommand/MCL/DeviceTrack.h

```diff
@@ -86,9 +86,9 @@ class DeviceTrack : public GridTrack {
     if (!that->GridTrack::load_from_mem(col)) {
       return nullptr;
     }
-    if (that->active != track_type) {
+   /* if (that->active != track_type) {
```

Comment:
sigh. I should've told myself that a nullptr will backfire, someday.

Reply:
reverted this one.

this fix was elsewhere: 214ace599a9e401ebc7199d2bc0a094befae89ad